index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Home page</title>
+</head>
+<body>
+    <section class="sec1">
+        <div class="header">
+            <button class="btn1">
```

Comment:
- [ ] great job with the mobile menu. Please kindly ensure that it covers the whole page when it pops-up.
![Screenshot (278)_LI](https://user-images.githubusercontent.com/65298048/182483144-b0cf4369-ae5e-42ae-ac8e-f452b7d25444.jpg)